README.md

```diff
@@ -148,3 +148,34 @@ limit
 | 2023-01-19 01:50:19.259 | 404       | /imgs/gcp-cross-project-service-account       | Google Chrome |
 +-------------------------+-----------+-----------------------------------------------+---------------+
 ```
+
+## Development
+
+### Run tests
+
+```bash
+make test
+```
+
+### Run linter
+
+```bash
+make lint
+```
+
+### Generate docs
+
+```bash
+make gen-docs
+```
+
+### Release a new version
+
+1. Follow [this link](https://github.com/simpleanalytics/cq-source-simpleanalytics/releases/new) to draft a new release.
+2. Click `Choose a tag` and enter the new version number:
+   ![image](https://user-images.githubusercontent.com/26760571/219360662-0ad1f83d-84c9-47c8-afb9-fe774ce03dcc.png)
+3. Click `Create new tag: <version> on publish` assuming it's a new tag.
+4. Click `Generate release notes` to automatically generate release notes.
+5. Click `Publish release` to publish the release.
```

Comment:
I think it also works to simply push a new tag, right?

Reply:
I think so! I'm not sure how the changelog will look like when you do that. Maybe it's worth configuring it via https://goreleaser.com/customization/changelog/#changelog

Reply:
Ah OK it defaults to just a list of commits:
https://github.com/simpleanalytics/cq-source-simpleanalytics/releases/tag/v1.0.0

Reply:
Let me update the instructions

Reply:
Simplified in https://github.com/simpleanalytics/cq-source-simpleanalytics/pull/2/commits/cf30413bd3708da67281a278bd78e41c0b7bc1e5